v03_pipeline/lib/reference_data/cadd.py

```diff
@@ -0,0 +1,68 @@
+import tempfile
+import urllib
+
+import hail as hl
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+
+# adapted from download_and_create_reference_datasets/v02/hail_scripts/write_cadd_ht.py
+def import_cadd_table(
+    raw_dataset_path: str,
+    reference_genome: ReferenceGenome,
+):
+    column_names = {
+        'f0': 'chrom',
+        'f1': 'pos',
+        'f2': 'ref',
+        'f3': 'alt',
+        'f4': 'RawScore',
+        'f5': 'PHRED',
+    }
+    types = {'f0': hl.tstr, 'f1': hl.tint, 'f4': hl.tfloat32, 'f5': hl.tfloat32}
+
+    with tempfile.NamedTemporaryFile(suffix='.tsv.gz', delete=False) as tmp_file:
+        urllib.request.urlretrieve(raw_dataset_path, tmp_file.name)  # noqa: S310
```

Comment:
made some tweaks to the og script and I need to test it but the raw files are BIG